### PR TITLE
Task #151 feat: Show hash(md5 string) of username & email data in reports

### PR DESCRIPTION
**Pull request by vaibhavsTekdi**
Show hash(md5 string) of username & email data in reports.

### CODE REVIEW

File: tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php [hunk 1]
```diff
@@ -153,6 +142,10 @@ protected function addIndexerEntry($user)
 			$values[]  = $db->quote($value);
 		}
 
+		// Add username & email hash values
+		array_push($columns, 'username_hash', 'email_hash');
+		array_push($values, "'".md5($user['username'])."'", "'".md5($user['email'])."'");
```
Comment (manojLondhe) [inline tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php]: phpcs

Comment (vaibhavsTekdi) [reply tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php]: Fixed.

File: tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php [hunk 2]
```diff
@@ -153,6 +142,10 @@ protected function addIndexerEntry($user)
 			$values[]  = $db->quote($value);
 		}
 
+		// Add username & email hash values
+		array_push($columns, 'username_hash', 'email_hash');
+		array_push($values, " ' " . md5($user['username']) . " ' ", " ' " . md5($user['email']) . " ' ");
```
Comment (manojLondhe) [inline tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php]: change as
```array_push($values, "'" . md5($user['username']) . "'", "'" . md5($user['email']) . "'");```

Comment (vaibhavsTekdi) [reply tjreports/plugins/user/tjreportsindexer/tjreportsindexer.php]: Fixed.

**Pull request merged**